spec/JSON-RPC.md

```diff
@@ -1561,6 +1562,39 @@ No parameters
 
 [Back to **List of methods**](#list-of-methods)
 
+## chain_getPossibleAuthors
+Returns the list of the account that can generate the blocks at the given block number.
+
+It returns `null` if anyone can generate the block.
+Only PoW and Solo chains can return `null`, other chains never return it.
+
+The possible authors of the genesis block are always the list that contains only the author of the genesis block, regardless of the chain types.
```

Comment:
The possible authors of the genesis block are always in the list that contains only the author of the genesis block, regardless of the chain types.